sigstore/verify/models.py

```diff
@@ -438,3 +451,62 @@ def rekor_entry(self, client: RekorClient) -> LogEntry:
             raise InvalidRekorEntry
 
         return entry
+
+    def to_bundle(self) -> Bundle:
+        """Converts VerificationMaterials into a Bundle. Requires that
+        the VerificationMaterials have a Rekor entry loaded. This is
+        the reverse operation of VerificationMaterials.from_bundle()
+        """
+        if not self.has_rekor_entry:
+            raise ValueError("Must have Rekor entry before converting to a Bundle")
```

Comment:
Nitpick: exception type here:

```suggestion
            raise InvalidMaterials("Must have Rekor entry before converting to a Bundle")
```